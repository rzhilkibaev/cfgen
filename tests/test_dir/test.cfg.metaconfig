var_to_be_overriden = echo var_overriden
var_dependent_on_another = echo another_var_value_is_$var_to_be_overriden