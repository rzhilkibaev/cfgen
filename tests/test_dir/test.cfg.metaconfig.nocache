var_no_cache