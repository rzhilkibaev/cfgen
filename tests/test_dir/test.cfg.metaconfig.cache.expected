var_from_root_config = var_from_root_config
var_to_be_overriden = var_overriden
var_dependent_on_another = another_var_value_is_var_overriden
